src/Orleans.Reminders/ReminderService/LocalReminderService.cs

```diff
@@ -29,20 +34,21 @@ internal sealed class LocalReminderService : GrainService, IReminderService
         internal LocalReminderService(
             Silo silo,
             IReminderTable reminderTable,
-            TimeSpan initTimeout,
+            IOptions<ReminderOptions> reminderOptions,
```

Comment:
The service provider doesn't like this ctor.

I haven't addressed this yet,  had to work through other refactors.

Reply:
I wonder what it's missing. It wouldn't work before because of the `TimeSpan` not being injectable, but your change looks fine to me. Maybe it's a construction order issue (eg, a cycle involving `Silo`)

Reply:
fixed this. using `ILocalSiloDetails` and `IConsistentRingProvider` from DI instead of Silo